examples/observing_notebooks/CWFSAlignFocusScript.ipynb

```diff
@@ -0,0 +1,280 @@
+{
```

Comment:
<p>and not apply the correction(s)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-ts/ts_notebooks/pull/42/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>